docs/HowToChooseTuner.md

```diff
@@ -171,6 +171,31 @@ _Usage_:
       eta: 3
 ```
 
+<a name="NetworkMorphism"></a>
+**Network Morphism**
+
+[Network Morphism](7) provides functions to automatically search for architecture of deep learning models. Every child network inherits the knowledge from its parent network and morphs into diverse types of networks, including changes of depth, width and skip-connection. Next, it estimates the value of child network using the history architecture and metric pairs. Then it selects the most promising one to train. More detail can be referred to [here](../src/sdk/pynni/nni/networkmorphism_tuner/README.md). 
+
+_Suggested scenario_: It is suggested that you want to apply deep learning methods to your task (your own dataset) but you have no idea of how to choose or design a network. You modify the [example](../examples/trials/network_morphism/cifar10/cifar10_keras.py) to fit your own dataset and your own data augmentation method. Also you can change the batch size, learning rate or optimizer. It is feasible for different tasks to find a good network architecture. Now we only support the cv domain.
+
+_Usage_:
+```yaml
+  # config.yaml
+  tuner:
+    builtinTunerName: NetworkMorphism
+    classArgs:
+      #choice: maximize, minimize
+      optimize_mode: maximize
+      #for now, we only support cv domain
+      task: cv
+      #input image width
+      input_width: 32
+      #input image channel
+      input_channel: 3
+      #output nums

```

Comment:
number of classes? would be better to make it more clear

Reply:
Ok. Thank you for your advice.